### PR TITLE
bugfix: missing `clear` table before to reuse table.

### PR DESCRIPTION
**Pull request by membphis**

### CODE REVIEW

**Comment by membphis** [comment]
this PR depends on https://github.com/apache/incubator-apisix/pull/1129

File: t/admin/routes.t [hunk 1]
```diff
@@ -1369,6 +1369,7 @@ passed
 location /t {
     content_by_lua_block {
         local t = require("lib.test_admin").test
+        local core = require("apisix.core")
```
Comment (moonming) [inline t/admin/routes.t]: why change this test case file?

Comment (membphis) [reply t/admin/routes.t]: that is a not related thing. Travis report failed sometimes because of this test case.

**Comment by moonming** [comment]
need to add more infomation about this bug: memory leak?

File: lua/apisix/core/response.lua [hunk 2]
```diff
@@ -37,10 +37,12 @@ do
     local idx = 1
 
 function resp_exit(code, ...)
+    clear_tab(t)
     idx = 0
 
     if code and type(code) ~= "number" then
-        insert_tab(t, code)
```
Comment (membphis) [inline lua/apisix/core/response.lua]: Always insert new data and then memory leak.

**Comment by membphis** [comment]
> need to add more infomation about this bug: memory leak?

please take a look at: https://github.com/apache/incubator-apisix/pull/1134/files#r380657179

**Pull request merged**